### PR TITLE
Lab02 Sandu Mihai mandatory assignments

### PR DESCRIPTION
**Pull request by melendus**
I added the mandatory assignments for the Lab 02, hope everything is okey, had some trouble with them but I think I managed to solve them in the end.

### CODE REVIEW

File: 30413/Mihai Alexandru Sandu/Lab02/tema1/3—f [hunk 1]
```diff
@@ -0,0 +1 @@
+none
```
Comment (bogdansalau) [inline 30413/Mihai Alexandru Sandu/Lab02/tema1/3—f]: Ce-i cu fisierele astea 3?

Comment (melendus) [reply 30413/Mihai Alexandru Sandu/Lab02/tema1/3—f]: Habar nu am cum au aparut, si eu m-am mirat de ce sunt acolo

Comment (bogdansalau) [reply 30413/Mihai Alexandru Sandu/Lab02/tema1/3—f]: Sterge-le te rog, trebuie sa pastram repo-ul curat.

Comment (melendus) [reply 30413/Mihai Alexandru Sandu/Lab02/tema1/3—f]: Okey, s-au creat cand rulam programul si nu stiu de ce.

**Comment by bogdansalau** [comment]
Mersi mult!

**Pull request merged**